### PR TITLE
Group shipments by TDL.

### PR DESCRIPTION
**Pull request by jim**
## Description

All shipments exist within a TDL. This patch exposes the TDL membership of shipments in the UI.

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Verification Steps

- [x] Tests all pass.
- [x] The UI is understandable.

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/155149514)

## Screenshots

screen shot 2018-02-23 at 2 48 58 pm

### CODE REVIEW

File: src/scenes/Shipments/index.jsx [hunk 1]
```diff
@@ -46,6 +46,25 @@ export class Shipments extends Component {
...
+      groups[shipment.traffic_distribution_list_id] =
+        groups[shipment.traffic_distribution_list_id] || [];
+      groups[shipment.traffic_distribution_list_id].push(shipment);
+
+      return groups;
+    }, {});
+
+    const cards = [];
+    for (let tdl in groupedShipments) {
+      const tdlID = tdl.substr(0, 6);
```
Comment (kilbergr) [inline src/scenes/Shipments/index.jsx]: Assume you're doing just for display purposes?

File: src/scenes/Shipments/index.jsx [hunk 2]
```diff
@@ -46,6 +46,25 @@ export class Shipments extends Component {
       );
     });
 
+    const groupedShipments = filteredShipments.reduce((groups, shipment) => {
```
Comment (kilbergr) [inline src/scenes/Shipments/index.jsx]: TODO: Move to reducer and memoize.
Inquire with Erin how should we allow users to sort.

File: src/scenes/Shipments/index.jsx (hunk 1 above)
Comment (jim) [reply src/scenes/Shipments/index.jsx]: Yeah, just so the TDLs can be differentiated. We should def. use other info (like source, destination regions) when it is available to show in the front end.

File: src/scenes/Shipments/index.jsx (hunk 2 above)
Comment (jim) [reply src/scenes/Shipments/index.jsx]: I can put this in a comment!

**Pull request merged**